test/testETHAPI.js

```diff
@@ -275,4 +278,36 @@ describe('ETH API', function() {
     })
   })
 
+  describe('getCode', function() {
+    it('contract with empty code', function(done) {
```

Comment:
'contract with empty code should be a string' or something, just write what to expect from the test